service-commands/src/commands/service-commands.json

```diff
@@ -63,7 +63,8 @@
     "port": 8899,
     "health_check_endpoint": "",
     "up": [
-      "docker run -d --name solana -p 8899:8899/tcp -p 8900-8902:8900-8902/tcp --network audius_dev --entrypoint='' solanalabs/solana:v1.9.13 sh -c 'solana-test-validator --gossip-host solana --limit-ledger-size 1000000'"
+      "sudo mkdir -p /mnt/disks/solana-disk",
+      "docker run -d --name solana -p 8899:8899/tcp -p 8900-8902:8900-8902/tcp --network audius_dev --entrypoint='' -v \"/mnt/disks/solana-disk:/test-ledger\" solanalabs/solana:v1.9.13 sh -c 'solana-test-validator --gossip-host solana --limit-ledger-size 200000000'"
```

Comment:
This gives us about 2 weeks of runtime:

https://github.com/solana-labs/solana/blob/36167b032c03fc7d1d8c288bb621920aaf903311/core/src/ledger_cleanup_service.rs#L23-L34

Reply:
200MM here is 400GB of data. This will break us.

Reply:
We're adding a 500GB disk on create-instance.sh and sending all the validator data to be stored on /mnt instead of within /var/lib/docker/containers/*. This would ensure we have about 2 weeks of data while isolating solana-validator data to be on its own disk.

This will stop everyone's root directory from running out of space every few days.

Reply:
Forgot to mention to that you can login to the machine I have noted with the PR's description to see how the new mount points are working.